frontend/dockerfile/docs/reference.md

```diff
@@ -0,0 +1,2384 @@
+---
+title: Dockerfile reference
+description: "Dockerfiles use a simple DSL which allows you to automate the steps you would normally manually take to create an image."
+keywords: "build, dockerfile, reference"
+redirect_from:
+- /reference/builder/
+---
```

Comment:
If we want to be more flexible, we could decide to have a "stub" `.md` file in the docs repository that includes this file; the stub could then contain the docs.docker.com-specific metadata, keeping them out of this repository (it would allow updating the metadata without requiring us to make changes in the upstream BuildKit repository)

Reply:
Atm I [move the `reference.md` to `builder.md`](https://github.com/docker/docker.github.io/pull/14948/commits/a67f515c82b18562b401c080c642669dbd4030b3) on docs repo but I don't really like it. You're also right that frontmatter metadata should live in the docs repo for this case. So this stub should include the remote `reference.md` as I understand but then where to fetch it? In `_includes`?

Reply:
Hm yes, I think it expects it to be in includes, but we'll have to try if that works, because I recall that Jekyll doesn't parse markdown in includes (but perhaps I did that wrong when I tried)